pkg/kubelet/kubelet_node_status.go

```diff
@@ -315,7 +315,7 @@ func (kl *Kubelet) initialNode() (*v1.Node, error) {
 // It synchronizes node status to master, registering the kubelet first if
 // necessary.
 func (kl *Kubelet) syncNodeStatus() {
-	if kl.kubeClient == nil {
+	if kl.kubeClient == nil || kl.standaloneMode {
```

Comment:
why is this needed? I wouldn't expect to get a kubeClient if we're in standalone mode